ansible/playbooks/wiki/wiki.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Wiki.js Service
+After=network.target
+
+[Service]
+Type=forking
```

Comment:
Everything looks good, just curious about why Type=forking?  Wiki.js docs have it as Type=simple.   

https://docs.requarks.io/install/linux

not saying it's wrong and I assume it works.  I checked the groups and directories mentioned exist in the playbook.

Reply:
Ah, I hadn't seen those docs. Thanks for pointing that out! I think I'll change the service to be more inline with what they suggest.

To answer your question about why I did Type=forking:
I based the service file off of how the service is currently being manually started and stopped.  If you look at the `ExecStart` and `ExecStop` options, you'll see that I'm running `/usr/bin/node wiki [start|stop]` whereas the docs suggest `/usr/bin/node server` for `ExecStart`.  Unlike the `server` program, the `wiki` program is itself a process manager ([PM2](https://pm2.keymetrics.io/)) which would control the `server` process and auto-daemonizes on start, hence the forking type.

Reply:
LGTM